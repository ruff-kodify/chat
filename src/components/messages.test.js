import React from 'react';
import Messages from './messages';

test('it should render properly', () => {
  const wrapper = shallow(
    <Messages
      users={[
        {
          id: '1',
          name: 'Unknown'
        }
      ]}
      messages={[
        {
          id: '1',
          type: 'user',
          body: 'foo',
          senderId: '1',
        },
        {
          id: '2',
          type: 'user',
          body: 'bar',
          senderId: '1'
        }
      ]}
    />
  );
  expect(wrapper).toMatchSnapshot();
});
