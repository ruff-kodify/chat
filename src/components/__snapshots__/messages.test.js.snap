// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it should render properly 1`] = `
<div
  className="messages"
>
  <Message
    body="foo"
    id="1"
    key="1"
    senderId="1"
    sentAt={1528221843537}
    type="user"
    users={
      Array [
        Object {
          "id": "1",
          "name": "Unknown",
        },
      ]
    }
  />
  <Message
    body="bar"
    id="2"
    key="2"
    senderId="1"
    sentAt={1528221843537}
    type="user"
    users={
      Array [
        Object {
          "id": "1",
          "name": "Unknown",
        },
      ]
    }
  />
</div>
`;
